reference/commands/consumer/get.rst

```diff
@@ -58,7 +56,7 @@ Gets a file or list a directory of a given reference or package.
 
   .. code-block:: bash
 
-      $ conan get zlib/1.2.11@conan/stable -p 09512ff863f37e98ed748eadd9c6df3e4ea424a8
+      $ conan get zlib/1.2.11@conan/stable:09512ff863f37e98ed748eadd9c6df3e4ea424a8
```

Comment:
This is more readable :smile_cat: 

But, as `-p` is suppressed now, but still supported, so why not add a note about that? If I read this documentation, I'll understand it like a breaking change, where `-p` is no longer supported.

Reply:
It has been removed from the help command but the parameter is still usable. Maybe we should leave a deprecation notice with the example below. WDYT?

Reply:
A deprecation notice sounds good!

Reply:
a deprecation notice sounds good!